src/tuning/kernels/xgemm.cpp

```diff
@@ -63,7 +63,7 @@ class TuneXgemm {
   static double DefaultInfluenceGlobalPSO(){ return 0.1; }
   static double DefaultInfluenceLocalPSO(){ return 0.3; }
   static double DefaultInfluenceRandomPSO(){ return 0.6; }
-  static size_t DefaultHeuristic(){ return size_t{3};} // PSO
+  static size_t DefaultHeuristic(){ return (size_t)cltune::SearchMethod::PSO;} // PSO
```

Comment:
Actually my first thought was to do like that, but then I realised that in order to propagate further the SearchMethod enum, we need to include the cltune header in utilities and, as side effect, it will be included everywhere. Do you think it is better? 
Yes, I will drop the comment